src/common/Environment.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2020 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CreateConnectionTransport } from './ConnectionTransport.js';
+import fetch from 'node-fetch';
```

Comment:
I think we should do this inline like we do for the other types:

```
fetch?: typeof import('node-fetch') | null
```

Is there a reason you didn't do this - just checking I'm not missing something 👍

Reply:
No, I think the method you mentioned is more reasonable, I'll fix it.

Reply:
Yes, `typeof import('node-fetch')` is not the fetch function type.
Finally, I found the following method is ok:
```typescript
  fetch?: typeof import('node-fetch').default | null
```
I've made a fix commit.

Reply:
and, anyway, the types of `node-fetch.fetch` and browser `globalThis.fetch` are not compatible,
so I choose to disable typescript check in `initialize-web.ts`.